package.json

```diff
@@ -16,11 +16,11 @@
   },
   "dependencies": {
     "ast-types": "^0.8.12",
+    "babel": "^5.8.23",
```

Comment:
It'd be better for you to depend on the `babel-core` package. It's the exact same without the CLI tools.